docs/identities/get-started/index.mdx

```diff
@@ -0,0 +1,26 @@
+---
+id: index
+title: Identity management guide - Day 1 essentials
+sidebar_label: Overview
+---
+
+# Identity management Guide
+
+This guide shows how to implement basic authentication flows with Ory. It covers the essential setup and implementation details
+for adding registration, login, and session management to your application.
+
+## What you'll learn
+
+- How to set up browser-based authentication flows
+- How to implement registration and login forms
```

Comment:
Registration flow is actually missing from the guides?

Reply:
I had registration here because the auth flow UI has both registration and login. Isn't that enough to pass as "implementing registration?"

Reply:
Yeah it‘s true that there is a „sign up“ link under the login box, but if you want to directly trigger registration then it‘s not explained in the guide. But it is an essential flow and probably not a lot of text required? Maybe we can even combine it with login in one page because it is essentially the same, just a different link target